src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -497,15 +503,23 @@ private FurtherInformation getFurtherInformation(DQ dq, CaseData caseData) {
 
     private RequestedCourt getRequestedCourt(DQ dq) {
         RequestedCourt rc = dq.getRequestedCourt();
-        if (rc == null) {
+        if (rc != null && null != rc.getCaseLocation()) {
```

Comment:
can we have consistency in the null checks please so both null != x

Reply:
done